docs/app/documentation/component-docs/date-picker/examples/date-picker-default-formats-example.component.ts

```diff
@@ -0,0 +1,37 @@
+import { Component } from '@angular/core';
+import { FdDate } from '../../../../../../library/src/lib/calendar/models/fd-date';
+import { FdRangeDate } from '../../../../../../library/src/lib/calendar/models/fd-range-date';
+
+@Component({
+    selector: 'fd-date-picker-default-formats-example',
+    template: `
+        <fd-date-picker [(ngModel)]="date"
+                        placeholder="yyyy.mm.dd"
+                        [dateFormat]="'yyyy.mm.dd'">
```

Comment:
could you add an input where the user can set the format and the value can be bound to `[dateFormat]`? This way we can play with different variations.